UITests/Receipts/TransactionDetailTests.swift

```diff
@@ -0,0 +1,198 @@
+import XCTest
```

Comment:
This Test has been deleted from development branch. Why is this here?

Reply:
I removed this file.